hedera-mirror-rest/__tests__/viewmodel/topicMessageViewModel.test.js

```diff
@@ -0,0 +1,106 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+'use strict';
+
+const {TopicMessageViewModel} = require('../../viewmodel');
+
+describe('topicMessageViewModel tests', () => {
+  test('Basic topic message', () => {
+    const actual = new TopicMessageViewModel(buildDefaultTopicMessageRow(), '');
+    const expected = buildDefaultTopicMessageViewModel();
+    expect(actual).toEqual(expected);
+  });
+
+  test('Basic topic message with UTF-8', () => {
+    const actual = new TopicMessageViewModel(buildDefaultTopicMessageRow(), 'UTF-8');
+
+    const expected = buildDefaultTopicMessageViewModel();
+    expected.message = 'message';
+
+    expect(actual).toEqual(expected);
+  });
+
+  test('Chunk info with valid start timestamp and payer account id', () => {
+    const input = buildDefaultTopicMessageRow();
+    input.chunkNum = 1;
+    input.chunkTotal = 10;
+    input.validStartTimestamp = '1234567890000000000';
+    const actual = new TopicMessageViewModel(input, '');
```

Comment:
need one test case with explicit `base64` encoding

```suggestion
    const actual = new TopicMessageViewModel(input, 'base64');
```

Reply:
Done